papers/config.py

```diff
@@ -218,6 +219,7 @@ def _fmt_path(p):
         else:
             status = bcolors.WARNING+"(default, not installed)"+bcolors.ENDC
         lines.append(bcolors.BOLD+f'papers configuration {status}'+bcolors.ENDC)
+        lines.append(bcolors.BOLD+f'Version {__version__}'+bcolors.ENDC)
```

Comment:
Lower case version would be more harmonious with the rest.

Here I made a few tests for visual appearance. Also testing a short version v3.1 on the header line. 
I guess they're all OK. Having version on its own line (second set of tests) takes space but it is clear and clean, admittedly. I'm OK with it.
![image](https://user-images.githubusercontent.com/1402264/233956762-2a98e943-906a-4596-a837-b1a7d7c4623b.png)

Reply:
I went with the last one in https://github.com/boyanpenkov/papers/commit/2809f176791034adc7fb0f6eec5dfaee88a6eff3